src/bci_build/package/postgres/README.md.j2

```diff
@@ -0,0 +1,114 @@
+
+#### POSTGRES_HOST_AUTH_METHOD
+
+This optional variable can be used to control the `auth-method` for host connections for all databases, all users, and all addresses. If unspecified then `scram-sha-256` password authentication is used.
+
+On an uninitialized database, this will populate `pg_hba.conf` via this approximate line:
+
+`echo "host all all all $POSTGRES_HOST_AUTH_METHOD" >> pg_hba.conf`
+
+See the PostgreSQL documentation on [password Authentication](https://www.postgresql.org/docs/14/auth-password.html) and [pg_hba.conf](https://www.postgresql.org/docs/14/auth-pg-hba-conf.html) for more information about possible values and their meanings.
+
+**Note 1:** If you set `POSTGRES_HOST_AUTH_METHOD` to `trust`, then `POSTGRES_PASSWORD` is not required since it allows anyone to connect without a password.
+
+**Note 2:** If you set this to an alternative value, you might need additional `POSTGRES_INITDB_ARGS` for the database to initialize correctly.
+
+#### PGDATA
+
+The default value for this variable is `/var/lib/pgsql/data`. This optional variable can be used to define another location for the [database files](https://www.postgresql.org/docs/current/storage-file-layout.html).
+
+This is an environment variable that is not container specific. The entrypoint script takes it into account because the variable is used by the `postgres` server binary.
+
+#### Sensitive information
+
+As an alternative to passing sensitive information via environment variables, `_FILE` may be appended to `POSTGRES_INITDB_ARGS`, `POSTGRES_PASSWORD`, `POSTGRES_USER`, and `POSTGRES_DB` environment variables, causing the initialization script to load the values for those variables from files present in the container.
+
+### Health, liveness, and readiness
+
+There is one explicit health check added to the container image. This check executes the `pg_isready` for host `localhost` and port `5432`.
+
+The utility [pg_isread](https://www.postgresql.org/docs/current/app-pg-isready.html) checks the connection status of the server and the exit status specifies the result of the connection check.
+
+**Warning:** Scripts in `/docker-entrypoint-initdb.d` are only run if you start the container with an empty data directory. Any pre-existing database will be left untouched on container startup.
+
+**Note:** If one of your `/docker-entrypoint-initdb.d` scripts fails and the container restarts with the already initialized data directory, it will not continue to run your scripts.
+
+### Initialization scripts
+
+If you would like to do additional initialization in an image derived from this one, add one or more `*.sql`, `*.sql.gz`, or `*.sh` scripts under `/docker-entrypoint-initdb.d`. After the entrypoint calls `initdb` to create the default PostgreSQL user and database, it will run any `*.sql` files, run any executable `*.sh` scripts, and source any non-executable `*.sh` scripts found in that directory to do further initialization before starting the service.
```

Comment:
the sentence is:

> in an image derived from this one

it implies:

you create a new container image based on this container image, and this here means the PostgreSQL image, because you are reading this sentence in the registry.suse.com page for PostgreSQL.

perhaps we can write:

> in an image derived from the PostgreSQL image

would this fit better?

Reply:
Yes, you suggestion is a good fix.